bls/BLSPublicKey.h

```diff
@@ -56,6 +56,11 @@ class BLSPublicKey {
     bool VerifySigWithHelper( std::shared_ptr< std::array< uint8_t, 32 > > hash_ptr,
         std::shared_ptr< BLSSignature > sign_ptr, size_t _requiredSigners, size_t _totalSigners );
 
+    bool AggregatedVerifySig(
```

Comment:
Well technically this method is not static. I used the same method signature as the other verification methods, with the exception of having vectors of signatures and hashes.
But I agree that the `_requiredSigners` and `_totalSigners` parameters are obsolete and could be removed from all the verification methods.

Reply:
However, I'm not to sure what additional constructor you think of?
From my perspective an aggregated signature with the corresponding aggregated hash is indistinguishable from a standard BLS signature for a given hash?

Reply:
I was thinking of constructor with inputs `(..., size_t totalSigners)` (without `requiredSigners`). but removing `totalSIgners, requiredSigners` from verification methods sounds good.

Reply:
Ok so I did three things now.
1.) I first removed the `requiredSigners` and `totalSigners` parameters from the verification methods
2.) I made the calls to the underlying `crypto::Bls`methods static. They already have been implemented as static methods, but where called through an object previously.
3.) I observed that the `requiredSigners` and `totalSigners` member variables of the `BLSPublicKey` class are basically never used anywhere in the library, except for some test-cases. So I removed them and adjusted the constructors accordingly, since only one of them actually needs these as a parameter.
I adjusted all calls to the constructors in codebase accordingly so everything works fine.
Nonetheless, this is a rather drastic change to the API, so let me know what you think of that.